### PR TITLE
ci: add some documentation

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## deepdetect-docker-build
+
+Build all docker images and push them on dockerhub.
+Keep in sync the dockerhub README with the GitHub README
+
+trigger every night on master branch
+trigger manually on release tag
+
+trigger
```
Comment (BynaryCobweb) [inline ci/README.md:43]: trigger what?

File: ci/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+$ ci/release.sh
+```
+
+If the result is OK, publish the release note on GitHub and push tags:
+
+```
+$ git push --tags
+```
+
+The script `ci/release.sh` update CHANGELOG.md, commit it, create a tag, and
```
Comment (BynaryCobweb) [inline ci/README.md:60]: updates

File: ci/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+# Building DeepDetect like the Jenkins  does:
+
+## Preparing the ubuntu images with all deps
+
+```
+$ docker build -t dd-dev -f ci/devel.Dockerfile --progress=plain .
+```
+
+### What to try another cuda, tensorrt, cudnn or ubuntu ?
```
Comment (BynaryCobweb) [inline ci/README.md:71]: Want?

**Review by BynaryCobweb — APPROVED** [review]

File: ci/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+```
+/var/lib/jenkins/jobs/deepdetect-prebuilt-cache/branches/master/builds/<BUILD ID>/archive/build/
+```
+
+* Triggered manually
+
+## deepdetect-prebuilt-cache-syncer
+
+Job that just keep in sync `/var/lib/jenkins/jobs/deepdetect-prebuilt-cache/` between ceres and netptune05
```
Comment (beniz) [inline ci/README.md:18]: replace machine names with 'between CI machines' ?

Comment (sileht) [reply ci/README.md]: done

**Review by beniz — APPROVED** [review]

**Pull request merged**